src/mlpack/methods/neighbor_search/kfn_main.cpp

```diff
@@ -194,10 +195,14 @@ int main(int argc, char *argv[])
       tree = KFNModel::R_PLUS_TREE;
     else if (treeType == "r-plus-plus")
       tree = KFNModel::R_PLUS_PLUS_TREE;
+    else if (treeType == "rp-tree-max")
+      tree = KFNModel::RP_TREE_MAX;
+    else if (treeType == "rp-tree-mean")
+      tree = KFNModel::RP_TREE_MEAN;
```

Comment:
Same here, I'd call one of these just the `rp-tree` and the other the `<split>-rp-tree`.